mender/templates/secret-s3-artifacts.yaml

```
@@ -14,5 +14,7 @@ data:
   AWS_URI: {{ .Values.global.s3.AWS_URI | b64enc }}
   AWS_BUCKET: {{ .Values.global.s3.AWS_BUCKET | b64enc }}
   AWS_REGION: {{ .Values.global.s3.AWS_REGION | b64enc }}
+{{- if not .Values.global.s3.useServiceAccount }}
   AWS_AUTH_KEY: {{ .Values.global.s3.AWS_ACCESS_KEY_ID | b64enc }}
   AWS_AUTH_SECRET: {{ .Values.global.s3.AWS_SECRET_ACCESS_KEY | b64enc }}
+{{- end }}
```

```suggestion
{{- if not .Values.global.s3.AWS_SERVICE_ACCOUNT_NAME }}
  AWS_AUTH_KEY: {{ .Values.global.s3.AWS_ACCESS_KEY_ID | b64enc }}
  AWS_AUTH_SECRET: {{ .Values.global.s3.AWS_SECRET_ACCESS_KEY | b64enc }}
{{- end }}
```

Pay attention to the missing new lline at the end of the file